crates/bevy_ui/src/ui_node.rs

```diff
@@ -14,7 +14,14 @@ use bevy_window::{PrimaryWindow, WindowRef};
 use std::num::{NonZeroI16, NonZeroU16};
 use thiserror::Error;
 
-/// Describes the size of a UI node
+/// Marker component for a UI node, which also provides the computed size of the node.
```

Comment:
marker component usually means tags and are zst

Reply:
Replaced "marker" with "base".